src/server/restOICP/v2.3.0/service/security/OICPEndpointSecurity.ts

```diff
@@ -0,0 +1,77 @@
+    const filteredRequest: HttpOICPEndpointsRequest = {} as HttpOICPEndpointsRequest;
+    // Set
+    filteredRequest.ID = sanitize(request.ID);
+    return filteredRequest;
+  }
+
+  static filterOicpEndpointRequestByID(request: any): string {
+    return sanitize(request.ID);
+  }
+
+  static filterOicpEndpointUpdateRequest(request: any): Partial<OICPEndpoint> {
+    const filteredRequest = OICPEndpointSecurity._filterOicpEndpointRequest(request);
+    filteredRequest.id = sanitize(request.id);
+    return filteredRequest;
+  }
+
+  static filterOicpEndpointCreateRequest(request: any): Partial<OICPEndpoint> {
+    return OICPEndpointSecurity._filterOicpEndpointRequest(request);
+  }
+
+  public static filterOicpEndpointsRequest(request: any): HttpOICPEndpointsRequest {
+    const filteredRequest: HttpOICPEndpointsRequest = {} as HttpOICPEndpointsRequest;
+    filteredRequest.Search = sanitize(request.Search);
+    UtilsSecurity.filterSkipAndLimit(request, filteredRequest);
+    UtilsSecurity.filterSort(request, filteredRequest);
+    return filteredRequest;
+  }
+
+  static filterOicpEndpointPingRequest(request: any): Partial<OICPEndpoint> {
+    const filteredRequest = OICPEndpointSecurity._filterOicpEndpointRequest(request);
+    filteredRequest.id = sanitize(request.id);
+    return filteredRequest;
+  }
+
+  static filterOicpEndpointTriggerJobRequest(request: any): Partial<OICPEndpoint> {
+    const filteredRequest = OICPEndpointSecurity._filterOicpEndpointRequest(request);
+    filteredRequest.id = sanitize(request.id);
+    return filteredRequest;
+  }
+
+  static filterOicpEndpointSendEVSEStatusesRequest(request: any): Partial<OICPEndpoint> {
+    const filteredRequest = OICPEndpointSecurity._filterOicpEndpointRequest(request);
+    filteredRequest.id = sanitize(request.id);
+    return filteredRequest;
+  }
+
+  static filterOicpEndpointSendEVSEsRequest(request: any): Partial<OICPEndpoint> {
+    const filteredRequest = OICPEndpointSecurity._filterOicpEndpointRequest(request);
+    filteredRequest.id = sanitize(request.id);
+    return filteredRequest;
+  }
+
+  static filterOicpEndpointRegisterRequest(request: any): Partial<OICPEndpoint> {
+    const filteredRequest = OICPEndpointSecurity._filterOicpEndpointRequest(request);
+    filteredRequest.id = sanitize(request.id);
+    return filteredRequest;
+  }
+
+  // eslint-disable-next-line no-unused-vars
+  static _filterOicpEndpointRequest(request: any): Partial<OICPEndpoint> {
+    const filteredRequest: Partial<OICPEndpoint> = {};
+    filteredRequest.name = sanitize(request.name);
+    filteredRequest.role = sanitize(request.role);
+    filteredRequest.baseUrl = sanitize(request.baseUrl);
+    filteredRequest.countryCode = sanitize(request.countryCode);
+    filteredRequest.partyId = sanitize(request.partyId);
+    filteredRequest.backgroundPatchJob = sanitize(request.backgroundPatchJob);
+    return filteredRequest;
+  }
+}
```

Comment:
I think there's a little misunderstanding here: the OICP REST API endpoints code just needs to be added to the current REST API directories structure. No need to create a specific directories structure.

Reply:
Yes, I agree.
I changed it